test.py

```diff
@@ -31,14 +75,20 @@ def rmtree(f):
 def print_linenum(signum, frame):
     print("Currently at line", frame.f_lineno)
 
-signal.signal(signal.SIGUSR1, print_linenum)
+if not isWin:
+    signal(signal.SIGUSR1, print_linenum)
```

Comment:
@AndreaBartolini the error you get is because Python module `signal` is called instead of function `signal.signal` of that module. No idea why this worked before, though...

Reply:
Oh and removing the extra indention level (2 spaces are used for indention) also prevents the error. I love white space breaking Python scripts 😆

Reply:
Well, I'll fix it

Reply:
Indeed in the original script the `signal.signal(signal.SIGUSR1, print_linenum)` call is (and was) used, probably I've deleted the first `signal.` when I modified the script...

Reply:
Fixed